prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchemaReader.kt

```diff
@@ -118,7 +120,8 @@ object ConfigSchemaReader : Logging {
         inputStream: InputStream,
         schemaClass: Class<out ConfigSchema<out ConfigSchemaElement>> = ConverterSchema::class.java
     ): ConfigSchema<*> {
-        val mapper = ObjectMapper(YAMLFactory())
+        val mapper = ObjectMapper(YAMLFactory()).registerKotlinModule()
```

Comment:
Some aspects of the Jackson deserialization didn't function as expected without an excessive number of annotations. Adding registerKotlinModule() resolved the issues.

Reply:
Just as a reference moving forward there is https://github.com/CDCgov/prime-reportstream/blob/bc40d2e04d422eb8519786ae4d0617524b67f213/prime-router/src/main/kotlin/common/JacksonMapperUtilities.kt#L56 that has a mapper set up with all the goodies.

Might be worth switching it out here or adding a similar utility class that lives in this package